io/net/ethtool_test.py.data/ethtool_test.yaml

```diff
@@ -15,10 +14,10 @@ Parameters: !mux
     RegisterDump:
         arg: -d
     EEPROMDump:
-        arg: -e
+       arg: -e
     RingParameter:
         arg: -g
-    ShowOffloadProtocolFeature:
+   ShowOffloadProtocolFeature:
```

Comment:
fix indent issue here:
Invalid multiplex file 'ethtool_test.py.data/ethtool_test.yaml': while parsing a block mapping

Reply:
unsupported on vnic:
Running 'ethtool -r env6 '
[stderr] Cannot restart autonegotiation: Operation not supported

Running 'ethtool -a env6 '
[stdout] Pause parameters for env6:
Command 'ethtool -a env6 ' finished with 76 after 0.00112390518188s
[stderr] Cannot get device pause settings: Operation not supported

Running 'ethtool -c env6 '
[stdout] Coalesce parameters for env6:
[stderr] Cannot get device coalesce settings: Operation not supported

Running 'ethtool -d env6 '
[stderr] Cannot get register dump: Operation not supported

Running 'ethtool -e env6 '
[stderr] Cannot get EEPROM data: Operation not supported

Running 'ethtool -p env6 3'
[stderr] Cannot identify NIC: Operation not supported

Running 'ethtool -t env6 '
[stderr] Cannot test: Operation not supported

Reply:
this will be a separate patch from me